src/main/java/system/components/ShapesGenerator.java

```diff
@@ -53,7 +53,7 @@ public static void generateShapes() throws Exception {
 
         int benchmarkNumber = 37;
 
-        for (int j = 480; j < 10000; j = j + 10) {
+        for (int j = 0; j < 10000; j = j + 10) {
```

Comment:
I changed this to be able to have some results on DBLP. Without it, I was receiving no output files
Will this affect the performance or the output ?